.github/workflows/mi.yml

```diff
@@ -3,7 +3,7 @@ name: MI
 on:
   push:
     branches:
-      - master
+      - main
```

Comment:
Just a small thing, could you also update the ReadME? Since it also mentions master branch?

Reply:
You are totally right! Will change it as well.